### PR TITLE
Create 2331.cpp

### PR DESCRIPTION
**Pull request by Lavanya-narang**
#### Checklist

- [x] Added description of change
- [x] Added file name matches [File name guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#File-Name-guidelines)
- [x] Added tests and example, test must pass
- [x] Relevant documentation/comments is changed or added
- [x] PR title follows semantic [commit guidelines](https://github.com/TheAlgorithms/C/blob/master/CONTRIBUTING.md#Commit-Guidelines)
- [x] Search previous suggestions before making a new one, as yours may be a duplicate.
- [x] I acknowledge that all my contributions will be made under the project's license.

Notes:

### CODE REVIEW

**Comment by Lavanya-narang** [comment]
A repo for leetcode solution 2331

**Comment by alexpantyukhin** [comment]
I'm not sure it's a C code

**Comment by Lavanya-narang** [comment]
But it's specifically mentioned to commit with . c file

File: leetcode/src/2331. Evaluate Boolean Binary Tree.cpp [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Solution {
```
Comment (tjgurwara99) [inline leetcode/src/2331. Evaluate Boolean Binary Tree.cpp:1]: This is not `C` - the `C` language doesn't even have the keyword `class` in it. What you have is a solution to the problem in `C++` and whether the source code is written inside the `.c` file or `.cpp` file has nothing to do with the language that is used.

**Comment by Lavanya-narang** [comment]
Thank you for the clarification.

**Comment by realstealthninja** [comment]
you should probably change this solution to either c code or try posting this to the c++ repository

**Pull request closed without merging**